README.md

```diff
@@ -167,52 +167,7 @@ For some known Ubuntu and macOS issues see the [pymaker](https://github.com/make
 
 ## Usage
 
-```
-usage: auction-keeper [-h] [--rpc-host RPC_HOST] [--rpc-port RPC_PORT]
-                      [--rpc-timeout RPC_TIMEOUT] --eth-from ETH_FROM
-                      [--eth-key [ETH_KEY [ETH_KEY ...]]] --addresses
-                      ADDRESSES [--type {flip,flap,flop}] [--ilk ILK]
-                      [--bid-only] [--vat-dai-target VAT_DAI_TARGET]
-                      [--keep-dai-in-vat-on-exit] [--keep-gem-in-vat-on-exit]
-                      --model MODEL [--debug]
-
-optional arguments:
-  -h, --help            show this help message and exit
-  --rpc-host RPC_HOST   JSON-RPC host (default: `localhost')
-  --rpc-port RPC_PORT   JSON-RPC port (default: `8545')
-  --rpc-timeout RPC_TIMEOUT
-                        JSON-RPC timeout (in seconds, default: 10)
-  --eth-from ETH_FROM   Ethereum account from which to send transactions
-  --eth-key [ETH_KEY [ETH_KEY ...]]
-                        Ethereum private key(s) to use (e.g.
-                        'key_file=aaa.json,pass_file=aaa.pass')
-  --type {flip,flap,flop}
-                        Auction type in which to participate
-  --ilk ILK             Name of the collateral type for a flip keeper (e.g.
-                        'ETH-B', 'ZRX-A')
-  --bid-only            Do not take opportunities to create new auctions
-  --max-auctions MAX_AUCTIONS
-                        Maximum number of auctions to simultaneously interact
-                        with, used to manage OS and hardware limitations
-  --min-flip-lot MIN_FLIP_LOT
-                        Minimum lot size to create or bid upon a flip auction
-  --vulcanize-endpoint VULCANIZE_ENDPOINT
-                        When specified, frob history will be queried from a
-                        VulcanizeDB lite node, reducing load on the Ethereum
-                        node for flip auctions
-  --from-block FROM_BLOCK
-                        Starting block from which to look at history (set to 
-                        block where MCD was deployed)
-  --vat-dai-target VAT_DAI_TARGET
-                        Amount of Dai to keep in the Vat contract (e.g. 2000)
-  --keep-dai-in-vat-on-exit
-                        Retain Dai in the Vat on exit, saving gas when
-                        restarting the keeper
-  --keep-gem-in-vat-on-exit
-                        Retain collateral in the Vat on exit
-  --model MODEL         Commandline to use in order to start the bidding model
-  --debug               Enable debug output
-```
+Run `bin/auction-keeper` without arguments to see an up-to-date list of arguments and usage information.
```

Comment:
I would instead propose to run `bin/auction-keeper -h`.

Reply:
done 👍